src/Util/Network.php

```diff
@@ -18,12 +18,13 @@
 class Network
 {
 	/**
-	 * @brief Curl wrapper
+	 * Curl wrapper
```

Comment:
AFAIK the `@brief` is used by the doxygen documentation builder to identify the summary of a function description. I think it should not be removed.

Reply:
It only is moved later in the comment. In the previous order, it would prevent some documentation parsers to even guess there's any text about the method/class